src/v2/guide/filters.md

```diff
@@ -62,20 +62,20 @@ Below is an example of our `capitalize` filter being used:
 </script>
 {% endraw %}
 
-The filter's function always receives the expression's value (the result of the former chain) as its first argument. In the above example, the `capitalize` filter function will receive the value of `message` as its argument.
+Filterfunktionen erhalten als erstes Argument immer den Ausdruckswert (d. h. das Ergebnis der vorausgehenden Kette). Im obigen Beispiel wird die `capitalize`-Filterfunktion als erstes Argument also den Wert von `message` erhalten.
```

Comment:
Mentioning a "vorausgehende Kette" (former chain) at this point is somewhat weird as this only gets introduced in the next paragraph. It would perhaps be better to say "das Ergebnis des Ausdrucks auf der linken Seite des Pipe-Symbols", but an improvement like this should probably go in the English original documentation first.